build.sbt

```diff
@@ -312,15 +313,15 @@ lazy val `kamon-kafka` = (project in file("instrumentation/kamon-kafka"))
   .disablePlugins(AssemblyPlugin)
   .enablePlugins(JavaAgent)
   .settings(instrumentationSettings)
+  .settings(crossScalaVersions += `scala_3_version`)
   .settings(
     libraryDependencies ++= Seq(
       kanelaAgent                 % "provided",
       "org.apache.kafka"          % "kafka-clients"     % "2.4.0" % "provided",
 
       scalatest                   % "test",
       logbackClassic              % "test",
-      "io.github.embeddedkafka"   %% "embedded-kafka"   % "2.4.1.1" % "test"
-    )
+      "io.github.embeddedkafka"   %% "embedded-kafka"   % "2.4.1.1" % "test" cross CrossVersion.for3Use2_13)
```

Comment:
Since there are no downstream dependencies on the test deps, for once this is fine 🤷